package.json

```diff
@@ -59,6 +59,8 @@
     "test": "mocha"
   },
   "dependencies": {
-    "bluebird": "^3.1.1"
+    "bluebird": "^3.1.1",
+    "liquid-node": "^2.6.1",
```

Comment:
Liquid node should be included in the dev dependencies instead. slm is already in the dev dependencies and should be removed.